internal/helpers/helpers.go

```diff
@@ -31,3 +35,20 @@ func GenerateDCNNode() []byte {
 
 	return b
 }
+
+func CreateMnemonic(addr []byte) (string, error) {
+	mn, err := mnemonic.EntropyToMnemonicThreeWords(addr)
+	if err != nil {
+		return "", err
+	}
+	name := strings.Join(mn, " ")
+
+	return name, nil
+}
+
+func IntToBytes(intVal int) []byte {
+	b := make([]byte, 32)
```

Comment:
It doesn't really matter, but note that the `32` in `Uint32` is bits, not bytes. Anyway, none of this matters because only the first 5 bytes are relevant to us.